velox/exec/tests/utils/DistributedPlanBuilder.cpp

```diff
@@ -108,6 +108,29 @@ core::PlanNodePtr DistributedPlanBuilder::shuffleResult(
   return std::move(planNode_);
 }
 
+core::PlanNodePtr DistributedPlanBuilder::broadcastResult() {
+  partitionedOutputBroadcast();
+  auto* output =
+      dynamic_cast<const core::PartitionedOutputNode*>(planNode_.get());
+  VELOX_CHECK_NOT_NULL(output);
+  const auto producerPrefix = current_->taskPrefix;
+  auto result = planNode_;
+  newFragment();
+  root_->stack_.pop_back();
```

Comment:
Can we check if stack is empty on dtor? And we should expect stack is not empty when calling non-result APIs from DistributedPlanBuilder? Thanks!